### ISSUE TITLE
Vim configuration requires token to be duplicated

### ISSUE THREAD

**Issue by typenil** [sourcery-ai/sourcery#89]
### Issue description or question
On my setup, the configuration specified in the [Vim Usage](https://github.com/sourcery-ai/sourcery/wiki/Vim-Usage) wiki results in the following error:

> Sourcery sign-up required. Click [here](https://sourcery.ai/download/) for a free token then enter it into the Sourcery section of the [settings](command:workbench.action.openSettings "Open Settings"

I have to specify the Sourcery token both in `initializationOptions` and in `settings`. If it's missing from either section, it generates the error above.

i.e.
```
{
    "languageserver": {
        "sourcery": {
            "command": "<Command to run Sourcery>",
            "args": [
                "lsp"
            ],
            "filetypes": [
                "python"
            ],
            "initializationOptions": {
                "token": "<Your Token Here>",
                "extension_version": "coc.vim",
                "editor_version": "vim"
            },
            "settings": {
              "sourcery": {
                "token": "<Your Token Here>",
                "metricsEnabled": true
              }
            }
        }
    }
}
```

### Sourcery Version

0.8.1

### Code editor or IDE name and version

NVIM v0.4.4

### OS name and version
Manjaro Linux Kernel 5.8.11-1

**Comment by brendanator** [comment]
Thanks for raising this. Have updated the wiki with your change. Will also look into removing the need for the token to be duplicated

**Comment by brendanator** [comment]
Fix identified, will be in the next release

**Comment by marcoaaguiar** [comment]
I'm not having this issue, is it because I'm using  0.8.0 instead of 0.8.1?

**Comment by Hellebore** [comment]
Fixed with release of 0.8.2.
